sdk/search/azure-search/src/test/java/com/azure/search/SearchTestBase.java

```diff
@@ -32,6 +31,10 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
I'll delete all the async tests and the base test classes in the next commit 😄

Reply:
Removed most async and base class combinations, I'll hold off on removing the rest for another PR since the few that are left contain more technical details I don't want to handle in this PR.